### ISSUE TITLE
[Feature] Routing between requests.

### ISSUE THREAD

**Issue by FoxSquared** [blinpete/wiki-graph#2]
It would be nice to have routing between requests.

As a user I want to be able to return back to my previous requests (or forth). Thereby restoring state of graph without making a request directly.

There are at least two cases of routing:
1. Routing between different requests - User requests new wiki page or language or both, and then wanted to return back.
2. Routing between nodes on the same graph - User made new root node using right mouse button and then wanted to return back. 

It would be nice to do it using the standard browser routing feature (buttons back and forth). But if it would be hard to implement. I think it could be enough to add new buttons and using internal page logic for this.

**Comment by blinpete** [comment]
Yep that's a nice feature! I am planning to implement this, just a bit low on time rn.

Thanks for your request 👍
